From eef152a3eaac5406e2b86d73951c8d64e4cdb177 Mon Sep 17 00:00:00 2001
From: JeanLuc Gernert <jangernert@gmail.com>
Date: Fri, 24 Feb 2017 18:12:00 +0100
Subject: [PATCH] fix #405

---
 plugins/share/Wallabag/WallabagAPI.vala | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/plugins/share/Wallabag/WallabagAPI.vala b/plugins/share/Wallabag/WallabagAPI.vala
index 80fb91f1..549c6afc 100644
--- a/plugins/share/Wallabag/WallabagAPI.vala
+++ b/plugins/share/Wallabag/WallabagAPI.vala
@@ -95,7 +95,7 @@ public class FeedReader.WallabagAPI : ShareAccountInterface, Peas.ExtensionBase
 		settings.set_string("client-secret", clientSecret);
 
 
-        var array = Settings.share("wallabag").get_strv("wallabag");
+        var array = Settings.share("wallabag").get_strv("account-ids");
 		foreach(string i in array)
 		{
 			if(i == id)
@@ -105,7 +105,7 @@ public class FeedReader.WallabagAPI : ShareAccountInterface, Peas.ExtensionBase
 			}
 		}
         array += id;
-		Settings.share("wallabag").set_strv("wallabag", array);
+		Settings.share("wallabag").set_strv("account-ids", array);
 
 
 		var pwSchema = new Secret.Schema ("org.gnome.feedreader.wallabag.password", Secret.SchemaFlags.NONE,
@@ -180,7 +180,7 @@ public class FeedReader.WallabagAPI : ShareAccountInterface, Peas.ExtensionBase
 			settings.reset(key);
 		}
 
-        var array = Settings.share("wallabag").get_strv("wallabag");
+        var array = Settings.share("wallabag").get_strv("account-ids");
     	string[] array2 = {};
 
     	foreach(string i in array)
@@ -188,7 +188,7 @@ public class FeedReader.WallabagAPI : ShareAccountInterface, Peas.ExtensionBase
 			if(i != id)
 				array2 += i;
 		}
-		Settings.share("wallabag").set_strv("wallabag", array2);
+		Settings.share("wallabag").set_strv("account-ids", array2);
 		deleteAccount(id);
 
         return true;
-- 
2.11.0

